src/site/content/en/blog/iframe-lazy-loading/index.md

```diff
@@ -0,0 +1,305 @@
+the user's viewport. Rather, it's only seen once they scroll further down the
+page. Despite this, users pay the cost of downloading data and costly
+JavaScript for each frame, even if they don't scroll to it.
+
+<figure class="w-figure">
+<img src="./iframe-lazyloading.png" alt="Data-savings from using iframe
+lazy-loading for an iframe. Eager loading pulls in 3MB in this example, while
+lazy-loading does not pull in this code until the user scrolls closer to the
+iframe.">
+</figure>
+
+Based off Chrome's research into [automatically lazy-loading offscreen iframes
+for Data Saver
+users](https://blog.chromium.org/2019/10/automatically-lazy-loading-offscreen.html),
+lazy-loading iframes could lead to 2-3% median data savings, 1-2% [First
+Contentful Paint](/fcp/) reductions at the median, and 2% [First Input
+Delay](/fid/) (FID) improvements at the 95th percentile.
+
+### How does native lazy-loading for iframes work?
+
+The `loading` attribute allows a browser to defer loading offscreen iframes and
+images until users scroll near them. `loading` supports three values:
+
+*   `lazy`: is a good candidate for lazy-loading.
+*   `eager`: is not a good candidate for lazy-loading. Load right away.
+*   `auto`: browser will determine whether or not to lazily load.
+
+{% Aside %}
+`auto` is currently a non-standard value, but is the default in Chrome today.
+Chrome intends on bringing a proposal for this value to the standards table.
+{% endAside %}
+
+Using the `loading` attribute on iframes works as follows:
+
+```html
+<!-- Lazy-load the iframe -->
+<iframe src="https://example.com" 
+        loading="lazy" 
+        width="600" 
+        height="400"></iframe>
+
+<!-- Eagerly load the iframe -->
+<iframe src="https://example.com" 
+        width="600" 
+        height="400"></iframe>
+
+<!-- or use loading="eager" to out of automatic 
+lazy-loading in Lite Mode -->
+<iframe src="https://example.com" 
+        loading="eager" 
+        width="600" 
+        height="400"></iframe>
+```
+
+Not specifying the attribute at all will have the same impact as explicitly
+eagerly loading the resource, except for [Lite
+Mode](https://blog.chromium.org/2019/04/data-saver-is-now-lite-mode.html)
+users, where Chrome will use the `auto` value to decide whether it should be
+lazy-loaded.
+
+If you need to _dynamically_ create iframes via JavaScript, setting
+`iframe.loading = 'lazy'` on the element is also
+[supported](https://bugs.chromium.org/p/chromium/issues/detail?id=993273):
+
+```js/2
+var iframe = document.createElement('iframe');
+iframe.src = 'https://example.com';
+iframe.loading = 'lazy';
+document.body.appendChild(iframe);
+```
+
+#### iframe-specific lazy-loading behavior
+
+The loading attribute affects iframes differently than images, depending on
+whether the iframe is hidden. (Hidden iframes are often used for analytics or
+communication purposes.) Chrome uses the following criteria to determine
+whether an iframe is hidden:
```

Comment:
Clarify that this applies to both `loading=lazy` and `loading=auto`. (Demo: http://software.hixie.ch/utilities/js/live-dom-viewer/saved/8306)

I think these heuristics should either be standardized or removed (for `loading=lazy`). cc @domfarolino

Reply:
Roger. Will add a note.